app/components/listing/ActionBar.scss

```
@@ -1,36 +1,39 @@
 @import "../../styles/utils/helpers";
 
 .action-sidebar {
-  margin: $padding-xlarge 0;
+  margin-bottom: $padding-xlarge 0;
```

```suggestion
  margin-bottom: $padding-xlarge;
```

I think you meant to drop the second number as well, since `margin-bottom` only takes a single value, whereas before we were using the `margin` shorthand for declaring vertical and horizontal margin with two values.

Actually, do you understand why we declare a bottom margin in the first place? There doesn't seem to be anything below the action sidebar, and it looks like we have an entirely separate component for the mobile version, so this wouldn't apply to that. If this is unused, then let's delete it altogether.

---

Ahhh good catch, there's no need for that rule. Thanks!